.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        language_version: python3
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        language_version: python3
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        exclude: ^(workaround)
+  - repo: https://github.com/econchick/interrogate
+    rev: 1.5.0
+    hooks:
+      - id: interrogate
+        args: [-vv]
+        exclude: ^(tests|workaround)
```

Comment:
I think this is neeeded: `exclude: ^(tests|workaround|versioneer.py)`. Otherwise, it finds a bunch of things in that script, which we don't care about.